docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,468 @@
+---
+title: Change Management in Apollo Federation
```

Comment:
Docs titles/headings are sentence-cased:
"Change management in Apollo Federation"
"Change management"